app/src/main/java/com/rentminder/Icon&Text.kt

```diff
@@ -100,25 +101,28 @@ fun OtherIconText() {
 
 @Composable
 fun TotalText() {
-    Text(
+    Row(verticalAlignment = Alignment.CenterVertically){Text(
         text = "Total:",
-        fontSize = (20.sp),
+        fontWeight = FontWeight.Bold,
+        fontSize = (26.sp),
         modifier = Modifier
             .padding(
                 top = 10.dp,
                 start = 15.dp
             )
-    )
+    )}
+
 }
 
 @Composable
 fun TotalPerPersonText() {
     Text(
         text = "Total Per Person:",
-        fontSize = (20.sp),
+        fontWeight = FontWeight.Bold,
```

Comment:
The text total per person will be bolded

Reply:
See comments above.